solution/test/test-largest-2.js

```diff
@@ -0,0 +1,11 @@
+const largest2 = require('../src/largest-2');
+const assert = require('assert');
+
+it("largest 2", () => {
+    const numbers = [1, 2, 3, 100, 4, 5, 6, 200];
+    let largest = largest2(numbers);
```

Comment:
How will the following test case affect your solution? ?

```
[1, 2, 5, 4, 3]
```

Reply:
Turns out my solution is wrong. Will fix.